news/_posts/2023-02-25-release-20230225.md

```diff
@@ -0,0 +1,26 @@
+---
+kind: article
+title: "Release 20230225"
+author: "PunkPun"
+created_at: 2023-02-25 18:00 +0000
+disqus_id: "release-20230225"
+permalink: "/news/release-20230225/"
+---
+
+Today we have a special occasion on our hands. We have been waiting for this moment for almost two years: A new stable OpenRA [Release 20230225]({{ '/download' | relative_url }}) is out now!
+
+<figure>
+  <video autoplay loop muted>
+    <source src="{{ '/images/news/20230225-2023.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20230225-2023.webm' | relative_url }}" type="video/webm">
+    <img src="{{ '/images/news/20230225-2023.png' | relative_url }}"/>
+  </video>
+</figure>
+
+A lot of features have built up over the two years. This release is packed to the brim with goodies small and large. Several major systems have been added and overhauled, a great deal of rougher edges smoothed out. For details see the previous playtest news posts [[one]({{ '/news/playtest-20221203/' | relative_url }}), [two]({{ '/news/playtest-20221223/' | relative_url }}), [three]({{ '/news/playtest-20230110/' | relative_url }}), [four]({{ '/news/playtest-20230218/' | relative_url }})] and, as always, for a full overview - [the full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/dc394e2f3532e7fe830118716ebdbaaa4e945eaf).
+
+But wait, there's more! This release brings major behind-the-scenes changes towards supporting the C&C Remastered Collection assets. This support is still incomplete, but works well enough that we are today also releasing a development preview of "Tiberian Dawn HD". It is a separate release and can be [downloaded here](https://github.com/OpenRA/TiberianDawnHD/releases/tag/release-20230225). This preview is multiplayer-compatible with the main 20230225 release, but performance, memory usage, and loading times have not been optimized, and do not reflect the expected final requirements. The C&C Remastered Collection must be installed through Steam or Origin, and the project README provides [installation instructions for macOS and Linux](https://github.com/OpenRA/TiberianDawnHD#asset-installation).
```

Comment:
```suggestion
But wait, there's more! This release brings major behind-the-scenes changes towards supporting the C&C Remastered Collection assets. This support is still incomplete, but works well enough that we are today also releasing a development preview of "Tiberian Dawn HD". It is a separate release and can be [downloaded here](https://github.com/OpenRA/TiberianDawnHD/releases/tag/release-20230225). This preview is multiplayer-compatible with the main 20230225 release, but performance, memory usage and loading times have not been optimized and do not reflect the expected final requirements. The C&C Remastered Collection must be installed through Steam or Origin, and the project README provides [installation instructions for macOS and Linux](https://github.com/OpenRA/TiberianDawnHD#asset-installation).
```
Extra commas.

Reply:
these are optional fwiw and pchote chose them 🤷